apps/extension/src/App/Accounts/AddAccount.tsx

```diff
@@ -137,19 +146,12 @@ const AddAccount: React.FC<Props> = ({ accounts, requester, setAccounts }) => {
             <p>HD Derivation Path</p>
             <Bip44PathContainer>
               <Bip44PathDelimiter>
-                {[bip44Prefix, coinType].join("/")}/
+                {[bip44Prefix, coinType, parentAccount].join("'/")}&apos;/
```

Comment:
nit: I feel it would be cleaner to use template string
```suggestion
                {`${bip44Prefix}'/${coinType}'/${parentAccount}'/`}
```